Makefile.toml

```diff
@@ -138,6 +139,12 @@ script = [
 '''
 ]
 
+[tasks.check-ts]
+description = "Runs tsc to validate TypeScript syntax."
+workspace = false
+install_script = ["(cd ./api_tests; yarn install; yarn run gen;)"]
```

Comment:
:mega: 
`yarn run gen` should be in the `postinstall` script I think. Why do we need to repeat it here?

Reply:
Oh, indeed, it was all messy before! Now cleaned up